### ISSUE TITLE
The question about conditional MI loss

### ISSUE THREAD

**Issue by PANPANKK** [pliang279/FactorCL#5]
Thanks for your exciting work! But what's the difference between self.infonce_x1y(self.linears_infonce_x1y(x1_embed), y) and self.infonce_x1x2_cond(torch.cat([self.linears_infonce_x1x2_cond[0](x1_embed), y_ohe], dim=1), 
                                              torch.cat([self.linears_infonce_x1x2_cond[1](x2_embed), y_ohe], dim=1)).  I mean why we need to use y but y_ohe?
